spec/mongoid/geospatial/helpers/spatial_spec.rb

```diff
@@ -12,10 +12,9 @@
 
     it 'should create correct indexes' do
       expect(Bar.collection.indexes.get(location: '2d'))
-        .to eq('key' => { 'location' => '2d' },
-               'name' => 'location_2d',
-               'ns' => 'mongoid_geo_test.bars',
-               'v' => 1)
```

Comment:
Looks like the newer version creates newer type of index, so we just stop caring about the version property.